src/styleguide/_patterns/04-templates/00-page-layouts/01-error-page.mustache

```diff
@@ -0,0 +1,11 @@
+{{> organisms-page-header }}
+<main class="grav-o-container">
+  <section class="grav-o-section--centered">
+    <h1>{{> atoms-ascii-art }} {{title}}</h1>
```

Comment:
I think we briefly talked about this, now that I see how it's been organised, I think it would be worth putting the ascii art in a separate container (a `<codeblock>` or a `<p>` with a special class?) and leave the error name in the `<h1>` so you don't have to do weird stuff.
You can always use the `@include typi('large-text')` if you need a larger font when styling

Reply:
I forgot that your ascii art is already in a separate tag. I'd just say that probably it's better to split them up.